tools/zypp-rpm/main.cc

```diff
@@ -270,6 +271,8 @@ int main( int, char ** )
   // the transaction data we will get in the callback
   TransactionData data { msg };
 
+  // do we care about knowing the public key?
```

Comment:
```suggestion
  // We may not need the public key if the package is secured by a strong checksum in signed metadata.
```

Reply:
This is for the libzypp side to decide, in zypp-rpm all we care about is if that switch is set or not. I'd suggest to move that comment to TargetImpl.cc instead.

Reply:
( maybe with reference here like: // see TargetImpl.cc : line )